Assets/Scripts/Core/GameManager.cs

```diff
@@ -0,0 +1,180 @@
+﻿using Assets.Scripts.Model;
+using UnityEngine;
+using UnityStandardAssets.Characters.FirstPerson;
+
+public class GameManager : MonoBehaviour {
+
+    public Configurator configurator;
+
+    public Player playerPrefab;
+
+    public static GameManager instance = null;
```

Comment:
Potrzebujemy to robić jako singleton? Nie wystarczy, żeby było DontDestroyOnLoad i potem pobierać ze sceny za pomocą GameObject.Find() albo GameObject.FindWithTag()?